editor/store/reducer.js

```diff
@@ -228,6 +228,15 @@ export const editor = flow( [
 		ignoreTypes: [ 'RECEIVE_BLOCKS', 'RESET_POST' ],
 	} ),
 ] )( {
+	autosave( state = false, action ) {
```

Comment:
Per https://github.com/WordPress/gutenberg/pull/6257#discussion_r183837142, I hesitate about including `autosave` here. Particularly, via `withChangeDetection`, `RESET_AUTOSAVE` would cause the post to be marked as dirty. I'm not sure if there are issues presently caused by this. (Should ideally be accounted for in E2E tests by #6209)

Reply:
Right! I moved this **back** out of the editor reducer section in https://github.com/WordPress/gutenberg/pull/6257/commits/724778061293c54cbb4db8416731bf3ea1314918